web/templates/first_topic_resolved_modal.hbs

```diff
@@ -0,0 +1,5 @@
+{{#tr}}
+You're marking the topic <b>{topic_name}</b> as resolved. This adds a ✔ at the beginning of the topic name to let everyone know that this conversation is done. <z-link>Learn more</z-link>
+{{#*inline "z-link"}}<a target="_blank" rel="noopener noreferrer" href="/help/resolve-a-topic">{{>
+    @partial-block}}</a>{{/inline}}
+{{/tr}}
```

Comment:
As a `confirm_dialog` template, this should be in `web/templates/confirm_dialog`.